### PR TITLE
Created common configuration file for CSM builds

### PR DESCRIPTION
**Pull request by tdawe**
# Description
Created common configuration file for CSM builds

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1031 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Tested with the karavi-metrics-powerflex repository. Verified the build was successful with the DEFAULT_BASEIMAGE. https://github.com/dell/karavi-metrics-powerflex/pull/158

### CODE REVIEW

**Review by rodrigobassil — APPROVED** [review]

**Comment by donatwork** [comment]
Why not ubi9?

File: config/csm-common.mk [hunk 1]
```diff
@@ -0,0 +1 @@
+DEFAULT_BASEIMAGE="registry.access.redhat.com/ubi8-micro@sha256:2c15d47056a573b30e608c4df43d20f79b064a8c10cb8b0c020912edf00c2399"
```
Comment (AronAtDell) [inline config/csm-common.mk:1]: This should be ubi 9, not so?

Comment (tdawe) [reply config/csm-common.mk:1]: Yes, I'll update it

**Comment by tdawe** [comment]
> Why not ubi9?
Fixed

File: config/csm-common.mk [hunk 2]
```diff
@@ -0,0 +1 @@
+DEFAULT_BASEIMAGE="registry.access.redhat.com/ubi9/ubi-micro@sha256:b7d8fd2840e92e8adc68414e13d859763ef33c3d4c4e27f910e939c00d642c29"
```
Comment (AronAtDell) [inline config/csm-common.mk:1]: Could we add a string that identifies the version in a comment? That way we can have everyone know what the sha pertains to. Please add a comment as well to ensure that the readable string is updated when a new version is updated.

Comment (tdawe) [reply config/csm-common.mk:4]: Updated with comments and details about what is required when we upgrade the base image

**Review by AronAtDell — APPROVED** [review]
Looks good to me. Thanks!

**Comment by sharmilarama** [comment]
How does this image get updated with new releases?

**Review by AronAtDell — APPROVED** [review]

**Comment by tdawe** [comment]
> How does this image get updated with new releases?

When a new UBI micro image is released, we just need to update this file in this repository. All other CSM repositories will pull this configuration file during their build which guarantees that we'll be using the same base image.

**Pull request merged**